nav2_bringup/README.md

```diff
@@ -3,30 +3,68 @@
 The `nav2_bringup` package is an example bringup system for navigation2 applications.
 
 ## Launch Navigation2 in simulation with Gazebo
- - Launch Gazebo and Rviz2
+Launch Gazebo and Rviz2
  
-```
-ros2 launch nav2_bringup gazebo_rviz2_launch.py world:=<full/path/to/gazebo.world>
-```
- - Launch Navigation2
+`ros2 launch nav2_bringup gazebo_rviz2_launch.py world:=<full/path/to/gazebo.world>`
+
+Launch your robot specific transforms
+
+Example: See [turtlebot3_gazebo](https://github.com/ROBOTIS-GIT/turtlebot3_simulations/tree/ros2/turtlebot3_gazebo) for details
+
+`ros2 launch turtlebot3_bringup turtlebot3_robot.launch.py`
+
+Set the tf publisher node to use simulation time or AMCL won't get the transforms correctly
+
+`ros2 param set /robot_state_publisher use_sim_time True`
+
+Launch map_server and AMCL
  
-```
-ros2 launch nav2_bringup nav2_bringup_launch.py map:=<full/path/to/map.yaml use_sim_time:=True 
-```
- - Set 'use\_sim\_time' parameter for static transforms
-* This is due to a bug in static\_transform\_publisher - [https://github.com/ros2/geometry2/issues/80](https://github.com/ros2/geometry2/issues/80)
+`ros2 launch nav2_bringup nav2_bringup_1st_launch.py map:=<full/path/to/map.yaml> use_sim_time:=True`
+
+In RVIZ:
+* Make sure all transforms from odom are present. (odom->base_link->base_scan)
+* Localize the robot using “2D Pose Estimate” button.
+
+Run the rest of the Navigation2 bringup
+
+`ros2 launch nav2_bringup nav2_bringup_2nd_launch.py`
+
+Set the World Model node to use simulation time
+
+`ros2 param set /world_model use_sim_time True`
```

Comment:
The global cost map doesn't get set. I still need to use `ros2 param set /global_costmap use_sim_time True`

Reply:
I have not needed to set that one but I'm testing again now with latest master branch

Reply:
OK, so I haven't seen exactly that /global_costmap needs to be set, but I have seen what I @crdelsey has also seen in that sometimes you have to set it twice. This whole setting of it is due to a bug in rclcpp I filed here:
https://github.com/ros2/rclcpp/issues/595

Anyway, until that is fixed, we have to set the use_sim_time variable after everything has been loaded. In some cases, it doesn't seem to take for all nodes, so we have to set it a 2nd time. 

In that case, we've seen you can simply set it again on /world_model and it should be OK. Again it's not perfect due to the bug above.